tests/kernel/trinity.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Trying to stress kernel with fuzz testing using trinity
+# Maintainer: Anton Smorodskyi<asmorodskyi@suse.com>
+
+use base "opensusebasetest";
+use testapi;
+use utils;
+use strict;
+use serial_terminal 'select_virtio_console';
+use upload_system_log;
+
+sub run {
+    my ($self) = @_;
+    select_virtio_console();
+    my $trinity     = 'trinity-1.8';
+    my $trinity_log = script_output("echo ~$testapi::username/trinity.log");
+    my $syscall_cnt=1000000;
+    my @grep_regex  = (
+        "Oops:",
+        "kernel BUG at",
+        "Kernel BUG at",
+        "WARNING: CPU",
+        "stack-protector: Kernel stack is corrupted",
+        "BUG: failure at",
+        "\\-\\[ cut here \\]\\-");
```

Comment:
You may get some false positives on Aarch64 about failed page faults/lookups because it prints a stack trace every time a user space program tries to access a page table entry which does not exist. That is, tries to access a memory address outside of any valid range, which happens a lot during fuzzing. However I don't think it is necessary to try fixing that now, just something to be aware of.